.github/workflows/os-zoo.yml

```diff
@@ -7,7 +7,7 @@
 
 name: OS Zoo CI
 
-on:
+on: [pull_request]
```

Comment:
do you want to change this?

Reply:
I presumed this was going to be undone on merge, but maybe there should be an explicit fixup commit added